client/components/editor-standalone/authoring-autosave.ts

```diff
@@ -91,11 +91,7 @@ export class AutoSaveHttp<T extends IBaseRestApiResponse> implements IAuthoringA
         return httpRequestJsonLocal<T>({
             method: 'GET',
             path: `/${this.resource}/${id}`,
-        }).then((res) => {
-            const result = this.modifyForClient(res);
-
-            return result;
-        });
+        }).then(this.modifyForClient);
```

Comment:
I'm just a little worried that `this.modifyForClient` inside `.then(` might lose the instance context (`this`). I'd suggest this below to be on the safe side:

```suggestion
        }).then((res) => this.modifyForClient(res));
```

Reply:
let me double check if that's any different than my version, from what I understood my version is just syntactic sugar for yours

Reply:
it actually makes a difference, but yeah double check it.